scripts/pre_commit_linter.py

```diff
@@ -310,6 +333,159 @@
+    },
+    {
+        'regexp': re.compile(r'urlunsplit'),
+        'message': 'Please use python_utils.url_unsplit().',
+        'excluded_files': ('python_utils.py'),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'parse_qs'),
+        'message': 'Please use python_utils.parse_query_string().',
+        'excluded_files': ('python_utils.py'),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'unquote\('),
+        'message': 'Please use python_utils.un_quote().',
+        'excluded_files': ('python_utils.py'),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'urljoin'),
+        'message': 'Please use python_utils.url_join().',
+        'excluded_files': ('python_utils.py'),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'urllib(2)?\..*Request\('),
+        'message': 'Please use python_utils.url_request().',
+        'excluded_files': ('python_utils.py', 'python_utils_test.py'),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\Winput\('),
+        'message': 'Please use python_utils.INPUT.',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'[^.|\w|\s]map\('),
+        'message': 'Please use python_utils.MAP.',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\Wnext\('),
+        'message': 'Please use python_utils.NEXT.',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'object\):'),
+        'message': 'Please use python_utils.OBJECT.',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\Wrange\('),
+        'message': 'Please use python_utils.RANGE.',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\Wround\('),
+        'message': 'Please use python_utils.ROUND.',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\Wstr\('),
+        'message': (
+            'Please try to use python_utils.STR. If you are getting an error, '
+            'use python_utils.convert_to_str() instead.'),
+        'excluded_files': ('python_utils.py'),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\Wzip\('),
+        'message': 'Please use python_utils.ZIP.',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'basestring'),
+        'message': 'Please use python_utils.BASESTRING.',
+        'excluded_files': ('python_utils.py'),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'__metaclass__'),
+        'message': 'Please use python_utils.with_metaclass().',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'iteritems'),
```

Comment:
Add check for `itervalues` and `iterkeys`.

Reply:
Done